crates/lunatic-control-axum/src/routes.rs

```diff
@@ -3,13 +3,14 @@ use std::sync::Arc;
 use axum::{
     body::Bytes,
     extract::DefaultBodyLimit,
-    routing::{get, post},
+    routing::{get, post, delete},
     Extension, Json, Router,
 };
 use lunatic_distributed::{
     control::{api::*, cert::TEST_ROOT_CERT},
     NodeInfo,
 };
+use lunatic_process::{ProcessName, ProcessRecord};
```

Comment:
Hmm not sure we should require lunatic-process crate just to get a string and node/process id. I think it's ok for the API to simply use String and define it's own {node_id, process_id} struct.

Reply:
Actually, we have merged submilisecond impl of ctrl srv which has lunatic-control crate with only types. We could add those types there...